corehq/pillows/user.py

```diff
@@ -135,6 +126,49 @@ def get_user_pillow(pillow_id='UserPillow', num_processes=1, process_num=0, **kw
     )
 
 
+def get_user_es_ucr_pillow(pillow_id='user-to-es-ucr-pillow', num_processes=1, process_num=0,
```

Comment:
rename to `get_user_pillow` and pillow_id to `user_pillow`

Reply:
The old pillow is called `get_user_pillow`, we can change the function name but not the pillow_id, since that's already in use.